### ISSUE TITLE
New gatsby version issue

### ISSUE THREAD

**Issue by theikid** [oorestisime/gatsby-source-instagram#188]
Hi there. I've switched to Gatsby V3 and the build failed because of this plugin : 

>9:31:04 AM: Cannot query field "allInstaNode" on type "Query".
>9:31:04 AM: If you don't expect "allInstaNode" to exist on the type "Query" it is most likely a typo.
>However, if you expect "allInstaNode" to exist there are a couple of solutions to common problems:

**Comment by oorestisime** [comment]
Hey i havent had the time yet to check what i need to do for gatsby v3. If anyone is willing to give it a go, PR is more than welcome!

**Comment by robSturcke** [comment]
@oorestisime I found some progress by running this locally with the following update to peer dependencies:

```
"peerDependencies": {
    "gatsby": "^3.3.1",
    "gatsby-source-filesystem": "^3.3.0"
  },
```

My logic being the first error in my production logs which is followed by the `allInstaNode` failure:
`warning " > gatsby-source-instagram@0.9.0" has incorrect peer dependency "gatsby-source-filesystem@^2.0.27".`

Hope that helps push this in the right direction.

**Comment by peterfortuin** [comment]
I have tried different version of gatsby 3 and gatsby-source-filesystem, but I have no luck yet.
I get the following error:

```
Could not dowcreate remote file noden, error is:  failed to process
<some url to instagram> after 3 attempts
```

**Comment by oorestisime** [comment]
Not in my immediate plans yet to work on this. if anyone is interested go for it

**Comment by DDeme** [comment]
I have the same issue. I don` t see any of properties in graphql schema.

**Comment by oorestisime** [comment]
@graysonhicks this will be fixed by your other PR right?

**Comment by graysonhicks** [comment]
Yes, this means the fetching from the API failed and no nodes were created.
